src/ci/bin/test.inc.sh

```diff
@@ -355,15 +355,11 @@ cromwell::private::create_build_variables() {
     backend_type="${backend_type#centaurWdlUpgrade}"
     backend_type="${backend_type#centaurHoricromtal}"
     backend_type="${backend_type#centaur}"
-    backend_type="${backend_type#conformance}"
     backend_type="$(echo "${backend_type}" | sed 's/\([A-Z]\)/_\1/g' | tr '[:upper:]' '[:lower:]' | cut -c 2-)"
     CROMWELL_BUILD_BACKEND_TYPE="${backend_type}"
 
-    if [[ "${CROMWELL_BUILD_TYPE}" == conformance* ]]; then
```

Comment:
TIL that "conformance" == CWL

Reply:
Yep, the CWL folks provide [standardized test cases](https://www.commonwl.org/v1.1/) for engines to run against, which is actually a pretty great idea.